src/build.py

```diff
@@ -896,7 +896,8 @@ def V8():
   buildbot.Step('V8')
   src_dir = work_dirs.GetV8()
   out_dir = os.path.join(src_dir, 'out.gn', 'x64.release')
-  proc.check_call(['vpython',
+  vpython = 'vpython' + '.bat' if IsWindows() else ''
```

Comment:
Can we add parens here to make the order of operations clearer?

Reply:
Done